styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java

```diff
@@ -84,7 +85,10 @@ void tick() {
 
       if (enabled.contains(workflow.id())) {
         try {
-          triggerListener.event(workflow, NATURAL_TRIGGER, decrementInstant(next, partitioning));
+          triggerListener.event(
+              workflow,
+              Trigger.natural(NATURAL_TRIGGER),
```

Comment:
It seems like the `natural` trigger should not have any `triggerId` field. It's always going to be this constant.